### PR TITLE
IElement tests

### LINKED ISSUE #6: Add tests for all public methods of IElement (including parent interfaces)

### PR DESCRIPTION
**Pull request by hsuboch**
closes #6

### CODE REVIEW

File: Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+using OpenQA.Selenium;
+
+namespace Aquality.WinAppDriver.Tests.Elements
+{
+    public class ElementTests : TestWithApplication
+    {
+        private readonly ITextBox rightArgumentTextBox = new CalculatorWindow().RightArgumentTextBox;
+        private const string ExpectedValue = "2";
+        private static readonly By ElementLocator = By.XPath("//*[@id='111111']");
+        const string ElementDescription = "Not present element";
```
Comment (mialeska) [inline Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs]: please add access modifier

File: Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+            Assert.NotNull(rightArgumentTextBox.GetElement());
+        }
+
+        [Test]
+        public void Should_ThrowNoSuchElementException_InGetElement_WhenElementIsNotPresent()
+        {
+            Assert.Throws<NoSuchElementException>(() => GetLabel().GetElement(TimeSpan.Zero));
+        }
+
+        private IElement GetLabel() => Factory.GetLabel(ElementLocator, ElementDescription);
```
Comment (mialeska) [inline Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs]: looks like a property, please, convert to property and put it among other properties upper

File: Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        [Test]
+        public void Should_GetCorrectText_FromEmptyField()
+        {
+            Assert.AreEqual(new CalculatorWindow().ResultsLabel.Text, string.Empty);
+        }
+
+        [Test]
+        public void Should_GetElement_ElementIsPresent()
```
Comment (mialeska) [inline Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs]: _WhenElementIsPresent

**Review by mialeska — CHANGES_REQUESTED** [review]

File: Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs (hunk 1 above)
Comment (hsuboch) [reply Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs]: fixed

File: Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs (hunk 2 above)
Comment (hsuboch) [reply Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs]: done

File: Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs (hunk 3 above)
Comment (hsuboch) [reply Aquality.WinAppDriver/tests/Aquality.WinAppDriver.Tests/Elements/ElementTests.cs]: fixed

**Review by mialeska — APPROVED** [review]

**Pull request merged**